### DISCUSSION TITLE
Abadonware?

### DISCUSSION THREAD

**Discussion by teward** [TheAssassin/AppImageLauncher#673] [Q&A] [answered]
The AppImageLauncher tool does not build in later versions of Ubuntu and has hard-failure cases during builds including in 2024 and 24.04 LTS.  There has also not been a build of appimagelauncher that has worked in modern Ubuntu releases, and no upstream movement since 2020, nor working nightly builds.

Has the software become abandonware?

**Comment by TheAssassin** [comment, marked as answer]
It has not. And please use the discussions for this.

The latest continuous release works fine in the latest Ubuntu. I use it every day.

I also have replied to this tons of times. Please use the search, folks...

**Reply by TheAssassin** [reply]
Answered in https://github.com/TheAssassin/AppImageLauncher/discussions/642.
